module-rt/src/bootstrap.rs

```diff
@@ -59,11 +59,9 @@ impl<T: UserModule + 'static> FoundryModule for ModuleContext<T> {
         self.user_context.replace(Arc::new(Mutex::new(module)));
```

Comment:
https://github.com/CodeChain-io/module-rt/issues/3
Since RTO context setup must be done concurrently with the other side, it should be done lazily not to cause a timeout in handshaking

Reply:
Could you add a simple description in the commit message? You can link the issue.

Reply:
ok